sockeye/arguments.py

```diff
@@ -821,10 +838,25 @@ def add_training_args(params):
                               help='How to normalize the loss. By default loss is normalized by the number '
                                    'of valid (non-PAD) tokens (%s).' % C.LOSS_NORM_VALID)
 
+    train_params.add_argument('--length-task',
+                              type=str,
+                              default=None,
+                              choices=[C.LENGTH_TASK_RATIO, C.LENGTH_TASK_LENGTH],
+                              help='If not None, adds an auxiliary task during training to predict source/target length ratios '
+                                    '(mean squared error loss), or absolute lengths (poisson) loss. Default %(default)s.')
+    train_params.add_argument('--length-task-weight',
+                              type=float_greater_or_equal(0.0),
+                              default=0.0,
```

Comment:
So this gives two ways of disabling: `--length-task` and the weight. How about setting it to a reasonable default so that when someone turns it on via `--length-task` it is not indirectly disabled through the weight.

Reply:
Will change to 1.0 (also there is a warning at [L144 in the sockeye/model.py](https://github.com/awslabs/sockeye/pull/626/files#diff-64c7821f0957b25e2f587b010d67e3afR144))